framework/src/media/Decoder.h

```diff
@@ -40,7 +40,7 @@ class Decoder
 
 private:
 #ifdef CONFIG_AUDIO_CODEC
-	//static int _configFunc(void *user_data, int audio_type, void *dec_ext);
+	// static int _configFunc(void *user_data, int audio_type, void *dec_ext);
```

Comment:
How about just deleting this line?
Is it just for backup? Then how about adding ToDo here..